### PR TITLE
feat: [FFM-5361]: update export

### PR DESCRIPTION
**Pull request by crystal-fong**
### Summary
Exported the `Description` component to be used within `EnvironmentDialog`

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by crystal-fong** [comment]
retrigger titlecheck

**Review by laura-mannus — APPROVED** [review]
LGTM

**Review by Kira-Jennings — APPROVED** [review]

File: src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const FFUIApp: FC = () => (
       useCDCreateEnvironment
     }}
     customHooks={{ useConfirmAction, useActiveEnvironment, useLicenseStore, useSyncedEnvironment }}
-    customComponents={{ RbacOptionsMenuButton, ContainerSpinner, SectionNoData }}
+    customComponents={{ RbacOptionsMenuButton, ContainerSpinner, Description }}
     customRoutes={routes}
     customUtils={{ NameSchema, getIdentifierFromName, IdentifierSchema }}
     customEnums={{ EnvironmentSDKKeyType, FeatureIdentifier, trackingConstants }}
```
Comment (knagurski) [inline src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx:49]: Can we kill off `EnvironmentSDKKeyType` while in the area?

Comment (crystal-fong) [reply src/modules/75-cf/pages/FFUIApp/FFUIApp.tsx]: my pleasure! removed

**Review by knagurski — APPROVED** [review]
🌟

**Review by laura-mannus — APPROVED** [review]

**Pull request merged**